sim2root/CoREASRawRoot/CorsikaInfoFuncs.py

```diff
@@ -7,6 +7,7 @@
 import io
 import numpy as np
 from datetime import datetime
+import pdb
```

Comment:
plese remove this import, is not being used.

Reply:
Fixed